scripts/run_kickstart_tests.sh

```diff
@@ -182,7 +182,10 @@ ISO_OS_NAME=$(echo "${output}" | grep 'NAME=')
 ISO_OS_NAME="${ISO_OS_NAME##NAME=}"
 ISO_OS_VERSION=$(echo "${output}" | grep 'VERSION=')
 ISO_OS_VERSION="${ISO_OS_VERSION##VERSION=}"
-
+ISO_PACKAGES=$(echo "${output}" | grep 'PACKAGES=')
+ISO_PACKAGES="${ISO_PACKAGES##PACKAGES=}"
+echo Saving list of installer image packages to /var/tmp/kstest-image-packages.log
+echo $ISO_PACKAGES | tr ' ' '\n' | tee /var/tmp/kstest-image-packages.log
```

Comment:
If I read this correctly then the list of packages will get printed to stdout. If this is correct I think it would spam the test results a lot wouldn't it? Isn't it enough to just store the list?

Reply:
If the tests pass, you do not need to worry about extra lines in the output, if there is a failure it might be handy to see the package versions right in the test output. I actually like it this way :-)

Reply:
I think in this case it is handy to have this in the log output without need of making sure we really pass / gather all the required logs (for example in GH actions). Also it is maybe more discoverable this way.